src/main/java/com/google/sps/servlets/ColumnsFromTablesServlet.java

```diff
@@ -60,7 +60,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
           for(int i = 0; i < listOfTables.length; i++){
               if(i != listOfTables.length-1){
                 constant.SELECTED_TABLES = "\'" + listOfTables[i] + "\'";
```

Comment:
When using the constants strings from Constants.java, we don't need to create an instance of the class. Instead, we can just import the static variables in that class, and use it in servlets. 

import static com.google.sps.servlets.Constants. GET_COLUMNS_FROM_TABLES;

And if they are constants, it probably doesn't make sense to modify them, like you discovered today, "constant.QUERY = ... " could be problematic. Just declare a new String and construct the query would be fine.

Reply:
Done